app/route_rangers_api/templates/Basetemplate_internal.html

```diff
@@ -1,173 +1,193 @@
                         </div>
                     </div>
                 </div>
             </div>
-        </footer>
-        <script>
-
-    // Initialize the map with Chicago as the center
-    var map = L.map('map').setView({{Coordinates}}, 13); // [latitude, longitude], zoom level
-    
-    // Add a tile layer
-    L.tileLayer('https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png', {
-        attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, &copy; <a href="https://carto.com/attribution">CARTO</a>',
-        maxZoom: 19
-    }).addTo(map);
-
-    // Loop through the stations and add a smaller marker for each
-    var stations_data = {{stations}};
-    stations_data.forEach(function(station) {
-        var marker = L.marker([station[0], station[1]]).addTo(map);
+        </div>
+    </footer>
+    <script>
+
+        // Initialize the map with Chicago as the center
+        var map = L.map('map').setView({{ Coordinates }}, 13); // [latitude, longitude], zoom level
+
+        // Add a tile layer
+        L.tileLayer('https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png', {
+            attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, &copy; <a href="https://carto.com/attribution">CARTO</a>',
+            maxZoom: 19
+        }).addTo(map);
+
+        // Loop through the stations and add a smaller marker for each
+        // but clustered!
+        var stations_data = {{ stations }};
+        var markers = L.markerClusterGroup({ disableClusteringAtZoom: 16 });
+
+
+        for (var i = 0; i < stations_data.length; i++) {
+            var station = stations_data[i];
+            var marker = L.marker([station[0], station[1]]);
+            markers.addLayer(marker);
+        };
+
+        map.addLayer(markers);
+
+
+        // // Custom icon for smaller markers
+        // var smallIcon = L.icon({
+        //     iconUrl: 'https://example.com/small-marker.png', // URL to a smaller icon image
+        //     iconSize: [12, 12], // Set the icon size you want
+        //     iconAnchor: [6, 6] // Adjust the anchor point if necessary
+        // });
+
+        // stations_data.forEach(function(station) {
+        //     var marker = L.marker([station[0], station[1]], {icon: smallIcon}).addTo(map);
+        // });
+
+        // add classes for mobile navigation toggling
+        var CSbody = document.querySelector('body');
+        const CSnavbarMenu = document.querySelector('#cs-navigation');
+        const CShamburgerMenu = document.querySelector('#cs-navigation .cs-toggle');
+
+        CShamburgerMenu.addEventListener('click', function () {
+            CShamburgerMenu.classList.toggle('cs-active');
+            CSnavbarMenu.classList.toggle('cs-active');
+            CSbody.classList.toggle('cs-open');
+            // run the function to check the aria-expanded value
+            ariaExpanded();
```

Comment:
I'm not sure why all of this is being flagged as changes because it's the same? Maybe formatting with Prettier will help with this

Reply:
My VSCode automatically removed one tab worth of indentation when I hit Save; should otherwise be the same